src/atoms/Button/Readme.md

```diff
@@ -1,36 +1,102 @@
-Default Button Example:
+## Button Variants
```

Comment:
I changed this page a bit to showcase all variants of the buttons. It looks like this now:
![image](https://user-images.githubusercontent.com/6582067/78153970-ad586300-7409-11ea-9e9c-6dd5bf0a9073.png)

(Note this is zoomed out super far to fit the whole thing, it looks better IRL)